pin.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# Set up flake registry to get o1js with all the submodules
+
+# Find the root of the o1js repo
+ROOT=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+pushd "$ROOT"
+# Update the submodules
+git submodule sync && git submodule update --init --recursive
+# Add the flake registry entry
+nix registry add o1js "git+file://$ROOT?submodules=1"
+# update mina input to local submodule
+nix flake update mina --override-input mina 'path:src/mina' --flake '.?submodules=1'
```

Comment:
Shouldn't it be the following? I'm not sure what the command does and why `--flake` argument is required

```suggestion
nix flake update o1js --override-input mina 'path:src/mina' --flake '.?submodules=1'
```